keyboards/40percentclub/5x5/5x5.h

```diff
@@ -37,7 +37,7 @@
     { K40, K41, K42, K43, K44, K45, K46, K47, K48, K49, ___, ___, ___, ___, ___} \
 }
 
-#define LAYOUT_ortho_5x15( \
+#define KEYMAP( \
```

Comment:
Hey Wanleg, we try to use the default standard names `LAYOUT_ortho_5x15` instead of `KEYMAP`. You can see the list of them in https://github.com/qmk/qmk_firmware/tree/master/layouts/default

This also helps when people use the community layouts to make use of the same layout on all their boards.

Unless this keymap is radically different from `LAYOUT_ortho_5x15`. We shouldn't change it. 

Also, we are moving from the term `KEYMAP` to `LAYOUT` because that's what the array represents.

Reply:
yeah, i totally get that, and i didn't really want to that if it can be avoided (but i kinda didn't want to put an alternate .h in my userspace either).
i was trying to figure out why `make xd75:wanleg` worked but `make 40percentclub/5x5:wanleg` didn't, and the difference was that `KEYMAP` bit. so i changed it and it compiled. do you know how i can do the same with `LAYOUT_ortho_5x15`?
what is `KEYMAP` doing that `LAYOUT_ortho_5x15` is not?

Reply:
reverted to original .h after applying drashna's suggestion.